src/main/resources/ASN.yaml

```diff
@@ -0,0 +1,33 @@
+---
+bulkloadconfig:
+  sampleFile : ASN.xml
+  numberOfFiles: 5
+  childNode: ASNDetail,LPN
+  numberOfMainNodes: 2
+  numberOfChildNodes: 2
+  numberOfSubChildNodes: 2
+  resultsFolderWithPath: "C:/performance-tests/target/jmeter/results/"
+xmlnodes:
+  -
+    name: ProductStatus
+    path: /tXML/Message/ASN/ASNDetail[$$]/InventoryAttributes/ProductStatus
+    value:
+      valueType: randomString
+  -
+    name: InstructionCode5
+    path: /tXML/Message/ASN/LPN[$$]/LPNDetail[$$$]/InstructionCode5
```

Comment:
Pls mention details about $$ and $$$ , when to use in the ReadMe . Also point read me to these sample configurations